scripts/makefile/tests.mk

```diff
@@ -86,12 +86,19 @@ else
 	@echo "scripts/makefile/status-report-validation.sh file does not exist"
 endif
 
-## Validate drupal-check
-drupalcheckval:
+## Validate drupal-rector
+drupalrectorval:
 	@echo "Drupal-check validation..."
 	$(call php, composer install -o)
-	$(call php, vendor/bin/drupal-check -V)
-	$(call php, vendor/bin/drupal-check -ad -vv -n --no-progress web/modules/custom/)
+	$(call php, vendor/bin/rector -V)
+	$(call php, vendor/bin/rector process --dry-run --no-progress-bar web/modules/custom web/themes/custom)
```

Comment:
If rector.yml file is required for script to run, best to wrap this around code to check file existence (check example in scripts/makefile/tests.mk)

Reply:
Added validation for file to exist

Reply:
Can't see where : o

Reply:
ah found but looks like misplaced : d

![image](https://user-images.githubusercontent.com/20738264/83269953-18cf6e80-a1c8-11ea-86cf-46d1c831d96e.png)

Reply:
Fixed